styleguide/source/_patterns/01-molecules/06-components/20-testimonial/_testimonial.scss

```diff
@@ -0,0 +1,78 @@
+.testimonial {
+  position: relative;
+  width: 100%;
+  max-width: 1200px;
+  margin-bottom: 24px;
+
+  @include breakpoint($bp-med) {
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: stretch;
+
+    &.testimonial-left {
+      flex-direction: row;
+    }
+  }
+}
+
+.testimonial_image {
+  display: flex;
+  flex-direction: column;
+
+  @include breakpoint($bp-med) {
+    min-width: 50%;
+    flex-basis: 50%;
+  }
+}
+
+.testimonial_image span {
+  margin: auto 0;
+  min-height: calc((100vw - 60px) / 2 * 3);
```

Comment:
Let's maybe eliminate the min-height and lock the mobile version at a height: 400px since that ensures we get the kind of portrait-y effect shown in the comp. All the larger breakpoints look fine they way they are.